File: backend/analytics-service/internal/entity/heatmap_cell.go
package entity

// HeatmapCell представляет одну ячейку тепловой карты: день недели, час и количество бронирований.
type HeatmapCell struct {
	Weekday uint8 // День недели (1-7, понедельник=0)
	Hour    uint8 // Час суток (0-23)
	Count   uint64 // Количество бронирований в данную ячейку
}

File: flutter-app/lib/core/models/heatmap_cell.dart
class HeatmapCell {
  final int weekday; // 0-6 или 1-7 (уточни backend)
  final int hour;    // 0-23
  final int count;

  HeatmapCell({
    required this.weekday,
    required this.hour,
    required this.count,
  });

  factory HeatmapCell.fromJson(Map<String, dynamic> json) => HeatmapCell(
    weekday: json['weekday'],
    hour: json['hour'],
    count: json['count'],
  );
}